setup.py

```diff
@@ -13,8 +13,8 @@
     include_package_data=True,
     install_requires=[
         'fedelemflowlist @ git+https://github.com/USEPA/Federal-LCA-Commons-Elementary-Flow-List.git@develop#egg=fedelemflowlist',
-        'esupy @ git+https://github.com/USEPA/esupy.git@develop#egg=esupy',
-        'StEWI @ git+https://github.com/USEPA/standardizedinventories.git@develop#egg=StEWI',
+        'esupy @ git+https://github.com/USEPA/esupy.git@urban_rural_classify#egg=esupy',
+        'StEWI @ git+https://github.com/USEPA/standardizedinventories.git@urban_rural_classify#egg=StEWI',
```

Comment:
if you have editable versions of packages installed locally, is it necessary to change these branches to urban_rural_classify?

Reply:
I see @bl-young changed it back

Reply:
yep - this isn't necessary anymore as there's nothing on that esupy branch needed in flowsa